<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="3">
       <widget class="QPushButton" name="divButton">
        <property name="text">
         <string>DIV</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="2">
       <widget class="QLineEdit" name="lineEdit2"/>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="multButton">
        <property name="text">
         <string>MULT</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="4">
       <widget class="QLineEdit" name="lineEdit3"/>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="subButton">
        <property name="text">
         <string>SUB</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLineEdit" name="lineEdit1"/>
      </item>
      <item row="3" column="0" colspan="4">
       <widget class="QPushButton" name="clearButton">
        <property name="text">
         <string>CLEAR</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="addButton">
        <property name="text">
         <string>ADD</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>690</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>addButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>add()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>340</y>
    </hint>
    <hint type="destinationlabel">
     <x>57</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>subButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>sub()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>340</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>multButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>mult()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>348</x>
     <y>338</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>divButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>div()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>517</x>
     <y>340</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>454</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>add()</slot>
  <slot>sub()</slot>
  <slot>mult()</slot>
  <slot>div()</slot>
  <slot>clear()</slot>
 </slots>
</ui>
